setup/air-gapped/index.md

```diff
@@ -117,15 +117,32 @@ platform images are hosted in Coder's Docker Hub repo.
    docker push my-registry.com/coderenvs/coder-service:<version>
    ```
 
+1. Create an `offline.values.yaml` file which includes the image paths for each
+   the Coder containers and proxy configuration (if necessary). Below is an
+   example:
+
+   ```yaml
+   coderd:
+      image: my-registry.com/coderenvs/coder-service:<version>
+   # Coder will use this proxy for all outbound HTTP/HTTPS connections
+   # such as when checking for updated images in the image registry.
+   # However, note that images are pulled from the Kubernetes container runtime,
+   # and may require a different setting.
+      proxy:
+         http: http://proxy.internal:8888
+         exempt: cluster.local
+   postgres:
+      default:
+         image: my-registry.com/coderenvs/timescale:<version>
```

Comment:
all the images should use a similar path as that's the most common case, suggest changing the envbox one to match